odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -390,8 +392,16 @@ class ImageConverter(models.AbstractModel):
 
     @api.model
     def _get_src_data_b64(self, value, options):
+        try:
+            img_b64 = base64.b64decode(value)
+        except (binascii.Error, ValueError):
+            raise ValueError("Invalid image content")
+
+        if img_b64 and guess_mimetype(img_b64, '') == 'image/webp':
```

Comment:
I'm not sure I understand.
the function _odoo_guess_mimetype uses an entry that checks for webp: (since commit d1292a96a62b18a88a852c69ed7732d16ae0344e)
```
    _Entry('image/webp', [b'RIFF'], [
        _check_webp,
    ]),
```